proposal.Rmd

```diff
@@ -0,0 +1,22 @@
+---
+title: "Proposal: shades of foundation offered by a variety of beauty brands around the world"
```

Comment:
I think the title is misleading because the makeup products aren't offered by beauty brands around the world. They are reviewed or recommended by the countries around the world. Also a shorter and catchier title may be better 😅

Reply:
Hmm, I am not sure about this - in the data README. it says:

`What is this?: Data representing the shades of foundation offered by a variety of beauty brands around the world.`

Could you confirm?